helm_deploy/values-dev.yaml

```diff
@@ -34,6 +34,12 @@ generic-service:
 
     REVIEWS_ENABLED: "true"
 
+  # TODO - move to parent `values.yaml` when we've requested and the HAAR team have created the client in preprod and prod
```

Comment:
If we had this in `values.yaml` before the client and secrets exist in preprod and prod, if we tried to deploy this build to preprod or prod it would not start up because the secrets do not exist yet